ibis/backends/pandas/tests/execution/test_arrays.py

```diff
@@ -30,22 +40,29 @@ def test_array_length(t, df):
 
 
 def test_array_length_scalar(client):
-    raw_value = [1, 2, 4]
-    value = ibis.literal(raw_value)
+    raw_value = np.array([1, 2, 4])
+    value = ibis.array(raw_value)
     expr = value.length()
     result = client.execute(expr)
     expected = len(raw_value)
     assert result == expected
 
 
 def test_array_collect(t, df):
+    expr = t.float64_with_zeros.collect()
+    result = expr.execute()
+    expected = np.array(df.float64_with_zeros)
+    assert type(result) == type(expected) and np.array_equal(result, expected)
```

Comment:
might be better to use `tm.assert_numpy_array_equal()` as this compares types (and nones as well)

Reply:
Ah hmm, `assert_numpy_array_equal` isn't exposed in `pandas.testing`. I can only access it through `pandas.util.testing` (deprecated) or `pandas._testing`. It seems like `assert_numpy_array_equal` isn't meant for non-internal use?

Reply:
import from pandas._testing is fine.

Reply:
Sounds good, I've changed them all to use `tm.assert_numpy_array_equal` now